console/console-init/ui/src/modules/iot-project-detail/IoTProjectDetailPage.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { Routes } from "./Routes";
+import { PageSection } from "@patternfly/react-core";
+import { ProjectNavigation } from "./components";
+
+export default function ProjectDetailPage() {
+  return (
+    <PageSection>
+      <ProjectNavigation activeItem="devices" />
```

Comment:
activeItem should be fetched from route params and by default it should be detail